packages/webapp/public/locales/en/translation.json

```diff
@@ -1427,6 +1427,18 @@
     "DESCRIPTION": "Someone will be in touch within 48 hours.",
     "TITLE": "Help request submitted"
   },
+  "REVENUE": {
+    "ADD_REVENUE": {
```

Comment:
Minor, but I'm wondering if we should reserve this translation string `REVENUE.ADD_REVENUE` for adding an actual revenue (aka sale), and make sure that `TYPE` makes it into this somewhere higher up? Like calling this `ADD_REVENUE_TYPE`?

Reply:
I followed what was done for expenses for better or worse. I think it is an offshoot of the "add revenue flow". Since you need to click add sale to get to this area.

I think ADD_SALE could be better for add sale. 

I would love to reorganize translations haha.